index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <main id="main">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+<h1 id="title">"Kaneki Kun"</h1>
+  <body>
+    "Hello World, we are software developers!"
```

Comment:
You may put this text inside an HTML element. You can use a h2 tag or even a div then you can handle it easily in css